.kitchen.vagrant.yml

```diff
@@ -1,44 +0,0 @@
----
-driver:
```

Comment:
well, I did not remove it, I tried to have only one .kitchen.yml:
 - DRY
 - currently kitchen-dokken does not work here

So the idea was to have only one kitchen.yml. Currently you will see locally the vagrant setup and it works without any `export KITCHEN_YAML` steps. In the CI you have the according DO env vars configured, so it works automatically with DO.

In the end its **one** `.kitchen.yml` and not two or three, where some of this files does not get updates usually.

I'm not sure yet how this should/could be implemented for chef-ssh-hardening, as somebody might use kitchen-dokken locally (what leads for my system to problems because of systemd) or some might use kitchen-vagrant locally. But this is a bit another discussion

Can you tell me where exactly do you see a problem with this approach? Do you have any suggestions how we can resolve that but in the same time keep one file?